cli/index.ts

```diff
@@ -405,8 +410,14 @@ const builtYargs = createYargsCli({
           compiledGraph.projectConfig.warehouse,
           compiledGraph.tests
         );
-        testResults.forEach(testResult => printTestResult(testResult));
-        return 0;
+        let allTestsSuccessful = true;
+        testResults.forEach(testResult => {
+          printTestResult(testResult);
+          if (testResult.successful === false) {
+            allTestsSuccessful = false;
+          }
+        });
+        return allTestsSuccessful ? 0 : 1;
```

Comment:
nit: i think this might be a little cleaner as `return testResults.some(testResult => !testResult.successful) ? 1 : 0`

Reply:
nit 2.0: `testResults.every(testResult => testResult.successful) ? 0 : 1`? One less operator to think through.